web/thesauruses/_meta/operators.json

```diff
@@ -46,6 +46,7 @@
       "bitwise_or": "Bitwise OR operator",
       "bitwise_or_assignment": "Bitwise OR and assignment operator",
       "bitwise_not": "Bitwise NOT operator",
+      "bitwise_not_assignment":"Bitwise NOT and assignment operator",
```

Comment:
`bitwise_not_assignment` doesn't really make sense since `bitwise_not` is not a binary operator.
The implementation below is just a composition of `bitwise_not` and the reglular assignment.

Reply:
Is it incorrect?
should i remove it or stay with it, what do you suggest?

Reply:
I can't think of a language that has a designated bitwise not & assign operator, nor can I imagine what it would do exept for maybe the same functionality as `bitwise_xor_assignment` but that's already there, so yes I would remove it.

Reply:
Hey, I just noticed that it already existed before i added it but with a different name, what to do now?

Reply:
I see you just moved it around, my bad I thought you added it. Mark it `not-implemented` like you did. We can deal with it in it's own PR.